source/Engine.cpp

```diff
@@ -619,6 +619,9 @@ void Engine::Step(bool isActive)
 		}
 	}
 
+	// Step the background to account for the current velocity and zoom.
+	GameData::StepBackground(centerVelocity, zoom.operator double());
```

Comment:
Will it not implicitly call the double conversion operator? If not, can we use a `static_cast<double>` or `double(zoom)`? This is not a normal way to write a conversion.

Reply:
It doesn't like the static cast. :(
It works implicitly, though.